ANTORA.md

```diff
@@ -1,82 +0,0 @@
-# Antora Integration
```

Comment:
Why is this (and other similar) files deleted? We'd definitely want to keep those.